indigo_api/serializers.py

```diff
@@ -556,8 +562,13 @@ def get_document_updated_at(self, obj):
 
 
 class CommencementSerializer(serializers.ModelSerializer):
-    commencing_title = serializers.CharField(source="commencing_work.title", default=None)
-    commencing_frbr_uri = serializers.CharField(source="commencing_work.frbr_uri", default=None)
+    """Details of a commencement event."""
+    commencing_title = serializers.CharField(source="commencing_work.title", default=None,
+                                             help_text="Title of the commencing work.")
+    commencing_frbr_uri = serializers.CharField(source="commencing_work.frbr_uri", default=None,
+                                                help_text="FRBR URI of the commencing work.")
+    provisions = serializers.SerializerMethodField(
```

Comment:
Changing this to a method means we can document it, and the type definition of the method will be picked up by drf-spectacular.